src/styles/styles/styles.scss

```diff
@@ -58,6 +58,10 @@ a.disabled {
   background: $button-primary;
 }
 
+a.button:hover {
+  color: #000;
```

Comment:
Please use variables for color. Also this is more like a generic selector that will be applied to all buttons. I believe this shouldn't be the case.  Can you be more specific regarding the selector?

Reply:
1. I used `#000`, because that is the format that used everywhere in the CSS file. I would also prefer the colour name, but I wanted to stay consistent with you guys.
2. I think this is a generic case, so I would leave it like this. If there is a more specific case, then we could overwrite it there. Am I wrong?

Reply:
1. You should define $black color in variable and start using it. Just like $white variable 
https://github.com/EMBL-EBI-SUBS/data-submissions-interface/blob/develop/src/styles/_variables.scss

Most of css in styles.scss is copies from EMBL-EBI Visual Framework which was using css instead of scss. This is why a lot of colors already exists are not following scss coding standard but this shouldn't be the case of any upcoming changes. 

2. In foundation there are many type of buttons (Primary, secondary, success, warning, alert..etc). There is also the hollow / border style button 

https://foundation.zurb.com/sites/docs/button.html

Each button has a different colors and style and I believe with using a generic black font that will be applied to .button class. This will affect all the button styles. 

Anyway, In my no need for css changes. just add the class "button primary" to the button instead of "button". This should add a hover effect.

Reply:
![screenshot 2018-10-10 at 4 42 01 pm](https://user-images.githubusercontent.com/2423218/46748461-85bc8a80-ccab-11e8-996c-5e765cf4eb05.png)

Reply:
OK, I will change the colour name to variable names, but then I am going to do it everywhere, because I don't like inconsistencies.
The same is true for the button styles, not every button which would look similar to the `Visit Your Dashboard` has the `button primary` class applied. This is inconsistent again.
The inconsistency could make confusion, so I would like to clean as much of it as we can.
I won't check the button style's everywhere with this ticket. I will open another ticket to fix it everywhere in the app.